src/test/java/com/camping/legacy/acceptance/reservation/ReservationAcceptanceTest.java

```diff
@@ -0,0 +1,115 @@
+package com.camping.legacy.acceptance.reservation;
+
+import com.camping.legacy.acceptance.reservation.builder.ReservationTestDataBuilder;
+import com.camping.legacy.acceptance.utils.AcceptanceTestBase;
+import com.camping.legacy.acceptance.utils.RequestAcceptanceFixture;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+import static com.camping.legacy.acceptance.utils.RequestAcceptanceFixture.*;
+import static com.camping.legacy.acceptance.utils.RequestAcceptanceFixture.updateReservation;
+import static com.camping.legacy.acceptance.utils.ResponseAcceptanceFixture.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReservationAcceptanceTest extends AcceptanceTestBase {
+    LocalDate today = LocalDate.now();
+    @DisplayName("유효한 정보를 통해 예약할 수 있다")
+    @Test
+    void reservationSuccessCase() {
+        /**
+         * 누가 : 예약하려는 사용자
+         * 언제 : 오늘
+         * 무엇을 : 30일내로 비어있는 날짜에 대한 예약을
+         * 왜 : 유효한 정보로 예약에 성공하기 위해
+         */
+        // given - 오늘 날짜를 기준으로 30일 이내의 날짜를 계산하고
+        ReservationTestDataBuilder builder = new ReservationTestDataBuilder();
+        Map<String, String> requestMap = builder.withSiteNumber("B-1").buildRequestMap();
+
+        // when - 30일 이내 날짜의 예약을 신청했을 때
+        ExtractableResponse<Response> response = createReservation(requestMap);
```

Comment:
지금도 충분히 좋은 테스트 코드입니다! 조금만 보충해보자면

given은 현재 사용자 입장에서 어떠한 행위를 했다고 보기 어려우니 when 구문에서 사용될 값을 준비하는 로직으로 볼 수 있는데요,
그런 관점에서 when 하나로 묶어주셔도 좋습니다만 현재의 기준도 좋습니다. 참고만 해주세요 :) 
그리고 만약에 주석으로 남겨주신 워딩으로 메서드를 분리해주신다면 주석 없이도 읽기 좋은 테스트가 될 것 같아요!

```suggestion
        val response = 30일_이내의_날짜의_예약을_신청했을_때()
```